history-deleter/history.css

```diff
@@ -1,4 +1,32 @@
 html, body {
-    margin: 0.2em;
-    width: 350px;
+    margin: 0.3em;
+    width: auto;
+    min-width: 250px;
+    max-width: 500px;
+    background-color: #f8f8f8;
 }
+
+div {
+	margin-left: 0.5em;
```

Comment:
minor nit, but these CSS properties in indented by different amounts, something consistent would be cool